foreman.te

```diff
@@ -353,6 +353,18 @@ systemd_config_generic_services(logrotate_t)
 manage_files_pattern(foreman_rails_t, system_cronjob_tmp_t, system_cronjob_tmp_t)
 manage_dirs_pattern(foreman_rails_t, system_cronjob_tmp_t, system_cronjob_tmp_t)
 
+######################################
+#
+# RPM temp workaround
+# https://community.theforeman.org/t/foreman-nightly-rpm-pipeline-946-failed/
+
+gen_require(`
+    type rpm_script_tmp_t;
+')
+
+manage_files_pattern(foreman_rails_t, rpm_script_tmp_t, rpm_script_tmp_t)
```

Comment:
https://selinuxproject.org/page/AVCRules suggests you can use `dontaudit` instead of `allow` in a rule. `manage_files_pattern` is defined as
```
define(`manage_files_pattern',`
        allow $1 $2:dir rw_dir_perms;
        allow $1 $3:file manage_file_perms;
')
```

So should we use:
```suggestion
dontaudit foreman_rails_t rpm_script_tmp_t:dir rw_dir_perms
dontaudit foreman_rails_t rpm_script_tmp_t:file manage_file_perms
```